CONTRIBUTING.md

```diff
@@ -1,41 +1,15 @@
-# Contribute to the documentation
-We really appreciate your support!
+# Helping out with Jamulus
 
-Please note that discussions should be kept in English. If you don't speak English fluently you can use a reasonably good online translator (e.g. [DeepL](https://www.deepl.com), [Reverso](https://www.reverso.net/text_translation.aspx), ...) which might be able to translate your language to English.
+We’d really appreciate your support and welcome help with proofreading, improvements, web design, and translation.
```

Comment:
Maybe make it more general and add something like "... welcome help and contribution. You could for example help with [...]". I doubt we can get all kinds of possible contribution here.